monix-eval/shared/src/main/scala/monix/eval/Coeval.scala

```diff
@@ -331,16 +332,33 @@ object Coeval {
   /** A `Coeval[Unit]` provided for convenience. */
   val unit: Coeval[Unit] = Now(())
 
-  /** Transforms a sequence of coevals into a coeval producing
-    * a list of gathered results.
+  /** Transforms a `TraversableOnce` of coevals into a coeval producing
+    * the same collection of gathered results.
     *
-    * For [[Coeval]] this has the same behavior as [[zipList]].
+    * It's a simple version of [[traverse]].
     */
-  def sequence[A](sources: Seq[Coeval[A]]): Coeval[List[A]] =
-    zipList(sources)
+  def sequence[A, M[X] <: TraversableOnce[X]](sources: M[Coeval[A]])
+                                             (implicit cbf: CanBuildFrom[M[Coeval[A]], A, M[A]]): Coeval[M[A]] = {
```

Comment:
Minor style issue, I know that IntelliJ IDEA indents code like this, but I don't like code indented that far to the right, because you end up with something looking like a grapes hanging of a branch of a tree or something. It's visually annoying imho. Shift this line to the left.